src/finite_gp_projection.jl

```diff
@@ -36,6 +30,11 @@ end
 Base.length(f::FiniteGP) = length(f.x)
 
 (f::AbstractGP)(x...) = FiniteGP(f, x...)
```

Comment:
Should it be 
```suggestion
(f::AbstractGP)(x...; kwargs...) = FiniteGP(f, x...; kwargs...)
```
or is it done automatically?

Reply:
No, it isn't done automatically (and hence one can't change `obsdim` with the official API - but this is not tested either...). However, since `FiniteGP` does not accept keyword arguments it is not really needed currently after applying the fixes and dispatches in this PR.